lib/driver.js

```diff
@@ -408,12 +409,11 @@ class XCUITestDriver extends BaseDriver {
   async executeCommand (cmd, ...args) {
     log.debug(`Executing command '${cmd}'`);
     if (cmd === 'receiveAsyncResponse') {
-      return await this.receiveAsyncResponse(...args);
+      return await this._executeCommandWithTimeout(B.promisify(this.receiveAsyncResponse), 'receiveAsyncResponse', ...args);
```

Comment:
because then I can use .timeout method of bluebird promise to execute it with a timeout. See _executeCommandWithTimeout for more details